### DISCUSSION TITLE
Successful installation but get no such file or directory

### DISCUSSION THREAD

**Discussion by letsroll1234** [Homebrew/discussions#770] [Getting started]
So I recently downloaded Homebrew and followed the installation steps. Now whenever I open the terminal I get `-bash: /bin/brew: No such file or directory` that open at the top three times. It is annoying and I can't seem to find a way to fix it. Any tips?

**Comment by carlocab** [comment]
You probably have a line in your `.zshrc` or `.bashrc` (or similar) that adds `brew` to your `PATH`. What does the line say currently?

**Reply by Mikelem** [reply]
I have two lines in my .zshrc file:
export PATH=/opt/homebrew/bin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/usr/local/sbin:/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/opt/X11/bin

export PATH=/opt/homebrew/bin:$PATH

brew runs, but after every brew command in Terminal I get:
`/opt/homebrew/bin/brew: line 11: cd: /usr/local/Homebrew/bin: No such file or directory`

**Reply by SMillerDev** [reply]
Do you use brew in the same terminal you installed brew in? Because it seems it was installed without Rosetta enabled and you're using it with Rosetta.

**Reply by Mikelem** [reply]
`% uname -m
arm64`

Yes, I'm running brew in the same terminal, but I'm not running rosetta. I'm pretty careful about not mixing them, and I've uninstalled and reinstalled Homebrew a few times.

I'm now thinking I should just wipe my system and do a clean install. This problem (as insignificant as it is) is a distant second to my bigger problem, not getting matplotlib to work. [Thanks for your help]

**Comment by gromgit** [comment]
What's the output of `ls -ld /opt/homebrew/bin/brew`?

**Reply by Mikelem** [reply]
-rwxr-xr-x  1 myname  wheel  3949 May 12 14:20 /opt/homebrew/bin/brew
